app/controllers/media_controller.rb

```diff
@@ -46,4 +56,12 @@ def media_params
   def load_media
     @media ||= StacksMediaStream.new(media_params)
   end
+
+  def remote_ip_addr
```

Comment:
this is the user IP, because `remote_ip_addr` is called when the token is being created, at which point the user is talking directly to stacks.

in `verify_token`, it's also the user IP, but it's given to stacks in a URL param, since wowza's making the request to stacks, at that point.

i can add a comment, that could use some clarification in the code.